workshops/templates/workshops/all_persons.html

```diff
@@ -17,14 +17,16 @@
 {% if all_persons %}
     <table class="table table-striped">
       <tr>
-        <th width="50">Instructor?</th>
+        <th width="25">DC?</th>
+        <th width="25">SWC?</th>
         <th>Name</th>
         <th>Email</th>
         <th class="additional-links"></th>
       </tr>
     {% for person in all_persons %}
       <tr>
-        <td class="text-center">{% if person.pk in instructors %}✓{% endif %}</td>
+        <td class="text-center">{% if person.pk in dc_instructors %}Ⓓ{% endif %}</td>
+        <td class="text-center">{% if person.pk in swc_instructors %}Ⓢ{% endif %}</td>
```

Comment:
I'd rather use 1 column and `bootstrap_tag` Django template tag defined in `workshops/templatetags/tags.py` which makes these nice labels:
![screenshot from 2016-05-05 07 47 38](https://cloud.githubusercontent.com/assets/72821/15036891/b265f21a-1295-11e6-9b21-61618c55d097.png)